README.md

```diff
@@ -112,69 +254,92 @@ synopsys_scan synopsys_security_product: "COVERITY"
+| `bitbucket_token` | The token can be configured in Jenkins **Global Configuration** or can be passed as **Environment Variable**. This is required if fixpr or prcomment is set true. <br> Example: `bitbucket_token: "${env.BITBUCKET_TOKEN}"` </br> | Optional             |
+
+### Synopsys Bridge Parameters
+| Input Parameter                     | Description                                                                                                                                                                                                                                                                                                                  |
+|-------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `synopsys_bridge_install_directory` | Provide a path, where you want to configure or already configured Synopsys Bridge. <br> [Note - If you don't provide any path, then by default configuration path will be considered as - $HOME/synopsys-bridge]. If the configured Synopsys Bridge is not the latest one, latest Synopsys Bridge version will be downloaded |
+| `synopsys_bridge_download_url`      | Provide URL to bridge zip file. If provided, Synopsys Bridge will be automatically downloaded and configured in the provided bridge- or default- path. <br> [Note - As per current behavior, when this value is provided, the bridge_path or default path will be cleaned first then download and configured all the time]   |
+| `synopsys_bridge_download_version`  | Provide bridge version. If provided, the specified version of Synopsys Bridge will be downloaded and configured.                                                                                                                                                                                                             |
+| `include_diagnostics`               | If this is set **true** then the detailed bridge logs will be shown in console and bridge diagnostics will be uploaded in Jenkins Archive Artifact.                                                                                                                                                                          |
+| `network_airgap`                    | When **network_airgap** is set **true**, <br/> - If Bridge exists in the default <Bridge install directory>, it is used. Otherwise, errors out. <br/> - If Bridge exists in the specified Bridge install directory, it is used. Otherwise, errors out. <br/>                                                                 |           
```

Comment:
@kishorikumar, have you any suggestions for the description of **network_airgap**.